stable_baselines3/common/policies.py

```diff
@@ -244,6 +244,18 @@ def _predict(self, observation: th.Tensor, deterministic: bool = False) -> th.Te
         :return: Taken action according to the policy
         """
 
+    @abstractmethod
```

Comment:
you can remove those, no? and use `train()` `eval()` in all other places

Reply:
what is the reasoning you would like to use `train()` and `eval()`? right now it is just one call in the base class.

Reply:
`train()` and `eval()` are defined by PyTorch so there should be no need of defining additional methods, no?

Reply:
ah oke :) wasn't really sure whether calling `train` on the policy affects all submodules. If I remember correctly, the feature extractor wasn't called, but maybe that was because of something else. I'll double check to make sure.

Reply:
Done. See commit 197936fb0dddceca799d0435dc59c62c369b4dc7

Reply:
well, it should work as train/eval are called recursively apparently: https://github.com/pytorch/pytorch/blob/52b8a52e4dfcbc074be9c743bde2a8f86aa3c0e1/torch/nn/modules/module.py#L1045-L1078